File: src/main/resources/static/js/carrental/service/carBandService.js
app.service('carBandService', function ($http) {

    this.getCarBandPage=function (curPage, size, searchCarBand) {
        return $http.post('/carBand/getCarBandPage?curPage='+curPage+'&size='+size, searchCarBand);
    }

    this.getCarBandById=function (id) {
        return $http.get('/carBand/getCarBandById?id='+id);
    }

    this.addCarBand=function (carBand) {
        return $http.post('/carBand/addCarBand', carBand);
    }

    this.updateCarBand=function (carBand) {
        return $http.post('/carBand/updateCarBand', carBand);
    }

    this.delCarBandById=function (id) {
        return $http.get('/carBand/delCarBandById?id='+id);
    }

    this.delCarBandByIds=function (ids) {
        return $http.get('/carBand/delCarBandByIds?ids='+ids);
    }
});

File: src/main/java/com/cfblj/carrental/service/CarBandService.java
package com.cfblj.carrental.service;

import com.baomidou.mybatisplus.extension.service.IService;
import com.cfblj.carrental.model.CarBand;
import com.cfblj.carrental.utils.Pages;

import java.util.List;

public interface CarBandService extends IService<CarBand> {
    void addCarBand(CarBand carBand);

    void updateCarBand(CarBand carBand);

    CarBand getCarBandById(String id);

    void delCarBandById(String id);

    void delCarBandByIds(String ids);

    Pages getCarBandPage(CarBand carBand, int curPage, int size);
}
